EPRN.PRNS.API/Controllers/PRNController.cs

```diff
@@ -164,6 +164,41 @@ public async Task<ActionResult> SaveDecemberWaste(int? id, bool decemberWaste)
             return Ok();
         }
 
+        [HttpGet]
+        [Route("Details")]
+        public async Task<IActionResult> GetPrnDetails(int id)
+        {
+            var prnDetailsDto = await _prnService.GetPrnDetails(id);
+            
+            return Ok(prnDetailsDto);
+        }
+
+        [HttpPost]
+        [Route("SentTo/{sentTo}")]
```

Comment:
The value of SentTo should come from the body of the request, not the url. This is because it's a string, and could be almost any length. URLs have a limit of the length, but body content doesn't

Reply:
Done.